float32 target_x
float32 target_y
float32 rotation
float32 gripper
float32 platform